apps/calendar/locales/calendar.en-US.properties

```diff
@@ -44,7 +44,7 @@ advanced-settings-add-account=Add Account
 field-user.placeholder=Username
 field-password.placeholder=Password
 field-full-url.placeholder=URL
-modify-account-setting-up=Please wait while your account is set up
+modify-account-setting-up=Setting up account
```

Comment:
I think you would need to modify it here as well: https://github.com/mozilla-b2g/gaia/blob/master/apps/calendar/elements/modify_account.html#L11